### ISSUE TITLE
Images in block nested vizy field not working

### ISSUE THREAD

**Issue by dGilli** [verbb/vizy#41]
**Description**
When adding an Image in a vizy block nested vizy field I can add alt text and title which are not saved. After inserting image alt text and title are not there and popup tooltip menu is not working. I can open back up the "Edit" modal but nothing happens on pressing "Update"

**Steps to reproduce**
1. Have nested vizy field in vizy block.
2. Add an image with alt text and title.
3. Try and edit image after inserting.

**Additional info**
- Plugin version: dev-craft-3 as 1.0.2
- Craft version: 3.6.16
- Multi-site: Yes

**Additional context**
Getting the following console error when trying to update image data after inserting it:
<img width="689" alt="Screenshot 2021-06-03 at 09 36 15" src="https://user-images.githubusercontent.com/14869578/120606249-43917a00-c44f-11eb-846a-8514048ff039.png">

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
